app/templates/components/modals/cfs-proposal-modal.hbs

```diff
@@ -8,13 +8,17 @@
     <p>{{t 'No existing sessions'}}</p>
   {{/if}}
   <div>
-    {{#link-to 'public.cfs.new-speaker' class='ui teal button' invokeAction=(action 'toggleView')}}
-      {{#if isNewSpeaker}}
+    {{#if isNewSpeaker}}
+      {{#link-to 'public.cfs.new-speaker' class='ui teal button' invokeAction=(action 'toggleView')}}
         {{t 'Add Speaker Details'}}
-      {{else}}
-        {{t 'Edit Speaker Details'}}
-      {{/if}}
-    {{/link-to}}
+      {{/link-to}}
+    {{else}}
+      {{#each data.userSpeaker as |speaker|}}
```

Comment:
Is it possible to have more than one speaker here?

Reply:
@pradeepgangwar no, but the query written returns a list of speakers because of the word query. Writing queryRecord was giving an error,